vendor.conf

```diff
@@ -11,6 +11,12 @@ github.com/rneugeba/iso9660wrap 4606f848a055435cdef85305960b0e1bb788d506
 github.com/satori/go.uuid b061729afc07e77a8aa4fad0a2fd840958f1942a
 github.com/surma/gocpio fcb68777e7dc4ea43ffce871b552c0d073c17495
 github.com/vmware/govmomi 6f8ebd89d521d9f9af7a6c2219c4deee511020dd
+github.com/Azure/azure-sdk-for-go 26132835cbefa2669a306b777f34b929b56aa0a2
+github.com/radu-matei/azure-sdk-for-go 3b12823551999669c9a325a32472508e0af7978e
```

Comment:
[This is the main reason.](https://github.com/linuxkit/linuxkit/pull/1939#discussion_r119320645)